packages/docs/package.json

```diff
@@ -31,7 +32,6 @@
     "eslint-plugin-import": "^2.28.1",
     "eslint-plugin-storybook": "^0.6.15",
     "html-format": "^1.1.2",
-    "lit": "^3.0.0",
```

Comment:
Not a big deal, but why was this moved to `dependencies`? As we will never ship @synergy-design-system/docs to npm, its not that big of a problem, but still I am not sure why this was added here?

Reply:
the linter flagged the previous location as wrong. That's why I changed it. But if not needed, I can change it back.
<img width="794" alt="Screenshot 2023-11-15 at 12 28 44" src="https://github.com/SickDesignSystem/synergy/assets/33674643/de694873-208f-4b55-b98f-1c430c6fa4f0">